### PR TITLE
Document using RichTextLabel's `meta_clicked` to handle clickable URLs

### PR DESCRIPTION
**Pull request by Calinou**
This information is already present in [BBCode in RichTextLabel](https://docs.godotengine.org/en/latest/tutorials/ui/bbcode_in_richtextlabel.html#doc-bbcode-in-richtextlabel-handling-url-tag-clicks) documentation, but since a lot of projects need this, it's worth making it available offline too in the editor class reference.

### CODE REVIEW

**Review by AThousandShips — APPROVED** [review]

File: doc/classes/RichTextLabel.xml [hunk 1]
```diff
@@ -674,7 +675,18 @@
...
+				Triggered when the user clicks on content between meta (URL) tags. If the meta is defined in BBCode, e.g. [code skip-lint][url={"key": "value"}]Text[/url][/code], then the parameter for this signal will always be a [String] type. If a particular type or an object is desired, the [method push_meta] method must be used to manually insert the data into the tag stack. Alternatively, you can convert the [String] input to the desired type based on its contents (such as calling [method JSON.parse] on it).
+				For example, the following method can be connected to [signal meta_clicked] to open clicked URLs using the user's default web browser:
+				[codeblocks]
+				[gdscript]
+				# This assumes RichTextLabel's `meta_clicked` signal was connected to
+				# the function below using the signal connection dialog.
+				func _richtextlabel_on_meta_clicked(meta):
+				    # `meta` is not guaranteed to be a String, so convert it to a String
+				    # to avoid script errors at run-time.
+				    OS.shell_open(str(meta))
```
Comment (Mickeon) [inline doc/classes/RichTextLabel.xml:687]: If it's not guaranteed to be a string it is not guaranteed to be a valid... "_shell thing_" to open, either. I feel like this is redundant in this case.

Comment (akien-mga) [reply doc/classes/RichTextLabel.xml:687]: Poke @Calinou

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 4.2.2.

**Pull request merged**